### PR TITLE
OSDOCS-332 - Add command to show PVC binding status

### PR DESCRIPTION
**Pull request by bobfuru**
[OSDOCS-332](https://issues.redhat.com/browse/OSDOCS-332)
@liangxia PTAL

### CODE REVIEW

File: modules/storage-persistent-storage-pv.adoc [hunk 1]
```diff
@@ -217,7 +217,11 @@ cluster.
 
 |===
 
-The CLI shows the name of the PVC bound to the PV.
+You can view the name of the PVC bound to the PV by running the following command:
+
+----
+$ oc get pvc <pv-claim>
```
Comment (liangxia) [inline modules/storage-persistent-storage-pv.adoc]: According to above explanation "You can view the name of the PVC bound to the PV", we want to get PVC name from the bound PV, so the command should be "oc get pv <pv-name>". E.g,
```
$ oc get pv pvc-028e3ee2-e497-4516-98a6-c72a89e88c4b
NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                                       STORAGECLASS               REASON   AGE
pvc-028e3ee2-e497-4516-98a6-c72a89e88c4b   5Gi        RWO            Delete           Bound    openshift-metering/hive-metastore-db-data   iscsi-targetd-vg-targetd            129m
```
We can get PVC name is hive-metastore-db-data in project openshift-metering.

Comment (bobfuru) [reply modules/storage-persistent-storage-pv.adoc]: Updated command. @liangxia please verify if LGTM.

**Comment by bobfuru** [comment]
@openshift/team-documentation PTAL

**Comment by bmcelvee** [comment]
LGTM!

**Pull request merged**